app/services/purecloud.js

```diff
@@ -92,6 +92,7 @@ export default Ember.Service.extend(Ember.Evented, {
 			.then(() => {
 				console.log('token destroyed');
 				window.localStorage.removeItem('purecloud_dev_tools_auth_auth_data');
+				window.localStorage.removeItem("environment");
```

Comment:
This should use `app/services/storage-service.js`.

I believe the line above it is forcibly scrubbing the remembered auth token. Verify that the item key it's using is correct (it doesn't look right off the top of my head). That one needs to use window.localStorage directly as it's reaching outside the scope of the app itself trying to clear something set by the SDK.